src/GraphQL.MicrosoftDI/ScopedFieldResolver.cs

```diff
@@ -9,6 +10,7 @@ namespace GraphQL.MicrosoftDI;
 /// to create a dependency injection scope. Then it calls a predefined <see cref="Func{T, TResult}"/>, passing the scoped service provider
 /// within <see cref="IResolveFieldContext.RequestServices"/>, and returns the result.
 /// </summary>
+[Obsolete("Please use ScopedFieldMiddleware instead. This class will be removed in a future release.")]
 public class ScopedFieldResolver<TReturnType> : FuncFieldResolver<TReturnType>
```

Comment:
ScopedFieldResolver is unused by GraphQL.MicrosoftDI now.  However if a developer did use it (for instance within GraphQL.Conventions), it will adjust the context accessor properly, which is pretty important.  Most of the other resolver wrappers are just type casting, and since the accessor only ever returns a non-strongly-typed `IResolveFieldContext`, it matters little whether it has the initial reference or the one with type casting.